jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/utils/ModalAssert.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.jdi.light.material.asserts.utils;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.material.elements.utils.Modal;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class ModalAssert extends UIAssert<ModalAssert, Modal> {
+
+    private String expectedTitle;
+    private String expectedDescription;
+
+    @JDIAction("Assert '{name}")
+    public ModalAssert verifyingTitle(String actualTitle) {
+        jdiAssert(element().verifyModalTitle(actualTitle), Matchers.is(expectedTitle));
+        return this;
+    }
+
+    @JDIAction("Assert '{name}")
+    public ModalAssert verifyingDescription(String actualDescription) {
+        jdiAssert(element().verifyModalDescription(actualDescription), Matchers.is(expectedDescription));
```

Comment:
Rewrite it:
        jdiAssert(element().verifyModalDescription(actualDescription), Matchers.is(true));